src/supertux/screen_manager.cpp

```diff
@@ -488,6 +489,13 @@ ScreenManager::process_events()
         {
           session->toggle_pause();
         }
+        break;
+
+      case SDL_DROPFILE:
+        char* filename = event.drop.file;
+        AddonManager::current()->install_addon_from_local_file(std::string(filename));
+        SDL_free(filename);
```

Comment:
This is part of `event.drop.file`, which is managed by SDL; are you sure it needs to be freed manually?

Reply:
According to https://wiki.libsdl.org/SDL_DropEvent it has to be freed manually.